torch/autograd/profiler.py

```diff
@@ -112,8 +112,8 @@ class profile:
     Args:
         enabled (bool, optional): Setting this to False makes this context manager a no-op.
 
-        use_cuda (bool, optional): Enables timing of CUDA events as well using the cudaEvent API.
```

Comment:
Could we keep use_cuda around for a little, and mark it as deprecating. It usually takes 1-2 releases to mark an API as deprecated (we could leave it in the args, but as a no-op).

Reply:
Sure. Thank you for suggestion.